pypeit/spectrographs/not_alfosc.py

```diff
@@ -111,21 +111,32 @@ def default_pypeit_par(cls):
         # Ignore PCA
         par['calibrations']['slitedges']['sync_predict'] = 'nearest'
         par['calibrations']['slitedges']['bound_detector'] = True
+        # Not a multi-slit instrument but flats are sometimes quite ugly leading to erroneous detection of multiple slits. Setting a higher edge_thresh seems to fail because of the bound_detector.
+        par['calibrations']['slitedges']['minimum_slit_gap'] = 15
 
         # Set pixel flat combination method
         par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
         # Wavelength calibration methods
         #par['calibrations']['wavelengths']['method'] = 'holy-grail'
         par['calibrations']['wavelengths']['method'] = 'full_template'
-        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'NeI']
+        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'NeI', 'ThAr']
```

Comment:
This makes me a little nervous... The ThAr list contains 17000 entries, while the NOT/ALFOSC ThAr list is only about 100 entries. My feeling is that you might be able to get away with adding `'ArI', 'ArII'` to this list, and possibly supplement this with a list of `'ThI'` lines based on the list provided on [NOT/ALFOSC page](http://www.not.iac.es/instruments/alfosc/lamps/thar.txt)?

Reply:
I've just tested it for grism 17 (which is the only one of the four that really uses the ThAr lamp) and you are right, we get a good, and less risky, wavelength solution just using ArI and ArII.  I will omit ThI for the moment but think about including a line list when I implement the rest of the grisms.